src/components/experiment-tracking/experiment-primary-toolbar/experiment-primary-toolbar.js

```diff
@@ -42,6 +42,7 @@ export const ExperimentPrimaryToolbar = ({
         labelText={`${selectedRunData?.bookmark ? 'Unbookmark' : 'Bookmark'}`}
         onClick={() => toggleBookmark()}
         visible={!enableComparisonView}
+        active={selectedRunData?.bookmark}
```

Comment:
Nit (coding style): I try to advise us to alphabetically sort component props, and also when destructuring and even CSS styles, where applicable. It makes readability easier when you know there's an order to things. There's a great [VSCode extension](https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines) that makes this easy, and would assume there's ways to do this in other IDEs as well.